### PR TITLE
feat: group commands in help

### LINKED ISSUE #2153: Group CLI help commands by category
**Describe the enhancement you'd like to see**
`testkube --help` lists all our help commands in alphabetical order and people have a hard time understanding where the command belongs to. 

- [ ] We'd like to cluster them by categories to help create a hierarchy
- [ ] We'd like to attach a link to `Docs & Support:   https://kubeshop.github.io/testkube/cli` at the bottom of the output

image

Figma:
https://www.figma.com/file/59vZTaJ6O2wTk0Qyqh2IJJ/Testkube-CLI?node-id=140%3A383

### PR DESCRIPTION
**Pull request by nicufk**
## Pull request description 

Closes #2153 

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by vLia — APPROVED** [review]
Looks good 👍

**Review by exu — APPROVED** [review]

File: cmd/kubectl-testkube/commands/help.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+			ui.Printf(fmt.Sprintf("%s %s", ui.White(RootCmd.Use), ui.LightGray("[flags]")))
+			ui.Printf("%s %s", ui.White(RootCmd.Use), ui.LightGray("[command]"))
+			ui.NL()
+			usage := helpMessageByGroups(RootCmd)
+			ui.Print(usage)
+			ui.Print(ui.LightGray("Flags"))
+			ui.Printf(RootCmd.Flags().FlagUsages())
+			ui.Print(ui.LightGray("Use \"kubectl testkube [command] --help\" for more information about a command."))
+			ui.NL()
+			ui.Printf("%s   %s", ui.LightGray("Docs & Support:"), ui.White("https://kubeshop.io/testkube"))
```
Comment (vsukhin) [inline cmd/kubectl-testkube/commands/help.go:40]: looks like a wrong link

File: cmd/kubectl-testkube/commands/help.go [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+	for k, _ := range groups {
+		groupNames = append(groupNames, k)
+	}
+	sort.Strings(groupNames)
+
+	buf := bytes.Buffer{}
+	for _, groupName := range groupNames {
+		commands := groups[groupName]
+
+		group := strings.Split(groupName, cmdGroupDelimiter)[1]
```
Comment (vsukhin) [inline cmd/kubectl-testkube/commands/help.go:80]: can it panic?

**Review by vsukhin — COMMENTED** [review]
a couple of questions

**Review by vsukhin — APPROVED** [review]

**Pull request merged**